src/main/webapp/pages/projects/settings/pages/members.html

```diff
@@ -6,10 +6,9 @@
 >
   <head>
     <title th:text="#{project.settings.page.title.members}">Title</title>
-    <link rel="stylesheet" th:href="@{/dist/css/project-users.bundle.css}" />
+    <webpacker:css entry="project-users"/>
     <script th:inline="javascript">
       window.PAGE = {
-        user: /*[[${session.user.getId()}]]*/ 1,
```

Comment:
Is this no longer used on the page?

Reply:
Good catch on that one, not sure what happened there.  Added back in 2ac6ecda1c3f8a6464dcaac95da55c5e3693bd14